pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -608,6 +641,233 @@ func TestReconcileClusterPool(t *testing.T) {
 			expectedAssignedClaims:   0,
 			expectedUnassignedClaims: 1,
 		},
+		{
+			name: "do not delete previously claimed clusters",
+			existing: []runtime.Object{
+				poolBuilder.Build(testcp.WithSize(3)),
+				unclaimedCDBuilder("c1").Build(testcd.Installed()),
+				unclaimedCDBuilder("c2").Build(testcd.Installed()),
+				unclaimedCDBuilder("c3").Build(),
+				cdBuilder("c4").Build(
+					testcd.WithClusterPoolReference(testNamespace, testLeasePoolName, "test-claim"),
+				),
+			},
+			expectedTotalClusters: 4,
+			expectedObservedSize:  3,
+			expectedObservedReady: 2,
+		},
+		{
+			name: "do not delete previously claimed clusters 2",
```

Comment:
Rather than appending a number I would prefer using the test name to note the specifics of that test vs the others.